### ISSUE TITLE
Can cleancalaccessrawdata be refactored to use lower-level command-line tools rather than rely on a slow Python loop through each file? 

### ISSUE THREAD

**Issue by palewire** [california-civic-data-coalition/django-calaccess-raw-data#1184]

**Comment by martinburch** [comment]
Could you point me to the slow loop that's involved here?

**Comment by gordonje** [comment]
[Line 126](https://github.com/california-civic-data-coalition/django-calaccess-raw-data/blob/master/calaccess_raw/management/commands/cleancalaccessrawfile.py#L126) in `cleancalaccessrawfile.py` is where we start looping over each line in a given .tsv file.

[Line 285](https://github.com/california-civic-data-coalition/django-calaccess-raw-data/blob/master/calaccess_raw/management/commands/updatecalaccessrawdata.py#L285) in `updatecalaccessrawdata.py` is where we loop over each .tsv file, calling the above management command.

Either of these what you are looking for @martinburch?

**Comment by martinburch** [comment]
Yes, I think the line iteration in cleancalaccessrawfile.py is where we can try applying the multiprocessing imap as discussed in slack.
